nupic/research/monitor_mixin/temporal_memory_monitor_mixin.py

```diff
@@ -383,10 +388,44 @@ def mmClearHistory(self):
 
     self._mmTraces["predictedCells"] = IndicesTrace(self, "predicted cells")
     self._mmTraces["activeColumns"] = IndicesTrace(self, "active columns")
+    self._mmTraces["activeCells"] = IndicesTrace(self, "active cells")
     self._mmTraces["predictiveCells"] = IndicesTrace(self, "predictive cells")
     self._mmTraces["numSegments"] = CountsTrace(self, "# segments")
     self._mmTraces["numSynapses"] = CountsTrace(self, "# synapses")
     self._mmTraces["sequenceLabels"] = StringsTrace(self, "sequence labels")
     self._mmTraces["resets"] = BoolsTrace(self, "resets")
-
     self._mmTransitionTracesStale = True
+
+
+  def mmGetCellActivityPlot(self, title="", showReset=False,
+                            resetShading=0.25, activityType="activeCells"):
+    """ Returns plot of the cell activity.
+
+    @param title (string) an optional title for the figure
+
+    @param showReset (boolean) if true, the first set of cell activities
+    after a reset will have a gray background
+
+    @param resetShading (float) If showReset is true, this float specifies the
+    intensity of the reset background with 0.0 being white and 1.0 being black
+
+    @param activityType (string) The type of cell activity to display. Valid
+    types include "activeCells", "predictiveCells", "predictedCells",
+    and "predictedActiveCells"
+
+    @return (Plot) plot
+    """
+    if activityType == "predictedActiveCells":
+      self._mmComputeTransitionTraces()
+
+    # If the trace contains ConnectionsCell, convert them to int
+    cellTrace = self._mmTraces[activityType].data
+    for i in xrange(len(cellTrace)):
+      if len(cellTrace[i]) > 0:
+        elem = next(iter(cellTrace[i]))
+        if isinstance(elem, ConnectionsCell):
```

Comment:
Here's my idea:
replace stuff inside for with
cellTrace[i] = self.getCellIndices(cellTrace[i])
What class should this new method go in, temporal_memory.py in nupic? or one of the others?